lib/Elastica/Query/ParentId.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Elastica\Query;
+
+final class ParentId extends AbstractQuery
```

Comment:
I usually apply `final` by default to classes that are not specifically designed to be extended. 
This avoids breaking an eventual child class when refactoring this class.
If you want i can remove `final` from this class.